orderbook/src/main.rs

```diff
@@ -123,6 +123,8 @@ async fn main() {
     .await
     .expect("failed to create gas price estimator");
 
+    // TODO - should we assert that token_filter doesn't overlap with base_tokens?
+    let deny_tokens = HashSet::from_iter(args.shared.token_filter);
```

Comment:
It is slightly unclear how the system should respond to denied base tokens given that they should be ignored, but are also being used in price estimation... I would think that these should not overlap, or that one of the two should trump the other. Looking for suggestions.

Reply:
I like asserting that they don't overlap.